webapp/src/ts/modules/reports/reports-content.component.html

```diff
@@ -40,7 +33,17 @@
             </span>
           </div>
           <mm-sender *ngIf="!summaries[$index]?.subject?.type" [message]="selection" [hideLineage]="true"></mm-sender>
-          <div test-id="form-title">{{summaries[$index] | title:forms}}</div>
+          <div test-id="form-title">
+            <span>{{ summaries[$index] | title:forms }}</span>
+            <div class="status" *ngIf="!validChecks[$index] || summaries[$index]?.verified !== undefined">
```

Comment:
Because now there's a fab button on the top right, we moved the verified badge next to the report's form title.